modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -822,6 +828,23 @@ static void configureInputInfo(const IE::InputInfo::Ptr& ii, const cv::GMetaArg
     }
 }
 
+static IE::PreProcessInfo configurePreProcInfo(const IE::InputInfo::CPtr& ii,
+                                               const cv::GMetaArg&        mm) {
+    IE::PreProcessInfo info;
+    if (cv::util::holds_alternative<cv::GFrameDesc>(mm)) {
+        auto desc = cv::util::get<cv::GFrameDesc>(mm);
+        if (desc.fmt == cv::MediaFormat::NV12) {
+            info.setColorFormat(IE::ColorFormat::NV12);
+        }
+    }
+    const auto layout = ii->getTensorDesc().getLayout();
+    if (layout == IE::Layout::NCHW ||
+        layout == IE::Layout::NHWC) {
+        info.setResizeAlgorithm(IE::RESIZE_BILINEAR);
```

Comment:
y don't we set resize in all other cases?

Reply:
I believe `resize` is relevant only for `NCHW`/`NHWC` layouts.
For instance how would you resize `NC` layout (it cause `segfault` :) )

Perhaps it possible for `CHW`/`HWC`, but I don't know such models which have these input layouts